librosa/core/constantq.py

```diff
@@ -656,7 +662,10 @@ def icqt(
 
     # truncate the cqt to max frames if helpful
```

Comment:
`# truncate the cqt to max frames if helpful` ? 🤔

Reply:
Looks like a comment that got misplaced after inserting some code by copy paste..